pkg/model/components/cilium.go

```diff
@@ -183,5 +183,16 @@ func (b *CiliumOptionsBuilder) BuildOptions(o *kops.Cluster) error {
 		}
 	}
 
+	gatewayAPI := c.GatewayAPI
+	if gatewayAPI != nil {
+		if gatewayAPI.Enabled == nil {
+			gatewayAPI.Enabled = fi.PtrTo(false)
```

Comment:
I think this should be true in this context.
```suggestion
			gatewayAPI.Enabled = fi.PtrTo(true)
```